openerp/modules/registry.py

```diff
@@ -172,6 +175,34 @@ def load(self, cr, module):
 
         return [self.models[m] for m in models_to_load]
 
+    def patch_migrate_9(self, cr):
```

Comment:
Can you put this in a new module odoo/openerp/openupgrade/openupgrade_loading_90.py?